pkg/config/utils.go

```diff
@@ -371,6 +371,17 @@ func processEnvVars(cliConfig *schema.CliConfiguration) error {
 		cliConfig.Settings.ListMergeStrategy = listMergeStrategy
 	}
 
+	versionEnabled := os.Getenv("ATMOS_VERSION_CHECK_ENABLED")
+	if len(versionEnabled) > 0 {
+		u.LogTrace(*cliConfig, fmt.Sprintf("Found ENV var ATMOS_VERSION_CHECK_ENABLED=%s", versionEnabled))
+		enabled, err := strconv.ParseBool(versionEnabled)
+		if err != nil {
+			u.LogWarning(*cliConfig, fmt.Sprintf("Invalid boolean value '%s' for ATMOS_VERSION_CHECK_ENABLED; using default.", versionEnabled))
```

Comment:
Also update this page: https://atmos.tools/cli/configuration/#environment-variables

Reply:
done